packages/skill-api/src/core/services/process-refund.ts

```diff
@@ -24,6 +29,12 @@ export async function stripeRefund({
       }
     }
 
+    const stripe = paymentOptions?.stripeCtx.stripe || defaultStripe
```

Comment:
these various skill-api endpoints are now defaulting to the configured stripe client from `PaymentOptions`, but since not all the apps are configuring that yet, I'm falling back to a manual import of the `stripe-sdk` stripe client